### ISSUE TITLE
Brave not saving passwords

### ISSUE THREAD

**Issue by MariuzM** [brave/brave-browser#4061]
## Description 

When clicking save passwords it's not saving them. 

## Expected result:
Save passwords? 

## Brave version (brave://version info)

Version 0.62.51 Chromium: 73.0.3683.103 (Official Build) (64-bit)

**Comment by rebron** [comment]
cc: @kjozwiak, @rebron to reproduce.

**Comment by coder006** [comment]
@rebron @kjozwiak the same is happening for me as well.
Browser Version: 
`Version 0.64.77 Chromium: 74.0.3729.169 (Official Build) (64-bit)`

Os Version:
  ```
System Version:	macOS 10.14.4 (18E226) (Mojave)
  Kernel Version:	Darwin 18.5.0
  Boot Volume:	Macintosh HD
  Boot Mode:	Normal
```
Although i am currently running the latest stable release, the issue is there from the last 3-4 months now.

Please let me know if you need any other details.

**Comment by sharifzadesina** [comment]
Same here.
It is always like this:
"Saved password will appear here"
But nothing appears.
![screenshot-settings-2019 06 16-03-17-50](https://user-images.githubusercontent.com/15300551/59557082-9dd86c80-8fe5-11e9-81c5-f8399a5b16cf.png)

**Comment by bcehprx** [comment]
Are you clearing browsing history on exit and not unchecking "Passwords and other sign-in data"?

**Comment by kjozwiak** [comment]
@MariuzM & @sharifzadesina are you two also on macOS?

**Comment by MariuzM** [comment]
Me yes

**Comment by bcehprx** [comment]
No. I'm on WIndows. I couldn't see a platform for Marius. I'd just figured out the reason my saved passwords were being deleted.

**Comment by sharifzadesina** [comment]
I am on linux.

**Comment by bcehprx** [comment]
And do you have this problem SIna?

**Comment by sharifzadesina** [comment]
@bcehprx Mine saves the passwords I can use them on websites.
But it doesn't show it.
I removed my post here and moved it to #3196.

**Comment by erezrokah** [comment]
Experiencing the same issue. Existing passwords are not shown anymore and clicking `save` doesn't add new ones.

The only solution I found is to create a new profile (Manage people->Add person) but then I lose all my existing passwords.

I'm using `Brave | 0.66.99 Chromium: 75.0.3770.100 (Official Build) (64-bit)` on `Ubuntu 18.04.2 LTS`

**Comment by NumDeP** [comment]
Could be a duplicate #624

**Comment by capouch** [comment]
I wasn't sure where to post this as mentioned above.  I just installed Brave Version 1.1.4 Chromium: 78.0.3904.97 on a fresh machine instance.  I already have the precise same version running on an older machine that had brave upgraded many, many times.

I am thinking something may have been bollixed up by one of the intermediary versions, because the old machine running this same version of the browser does appear to be storing the passwords but doesn't show any of them, and when autofill time comes there is a popup but none of the text in it is visible until you hover over it. . . 

But on this fresh machine instance, all seems to be working just fine.  I saved a bunch of passwords and then took the machine down and rebooted, and they survived the reboot.

**Comment by bsclifton** [comment]
Several +1's from https://github.com/brave/brave-browser/issues/5902

> 
>
> ## Description
> 
> Many users have reported that Brave will no longer save their login/password data. After entering credentials into a forum, they're offered to save the password via browser prompt, but the data never gets stored. Data saved in this way is not reflected in `Settings --> Autofill --> Passwords` either.
>
> ## Steps to Reproduce
> 
> **These are the steps that the users went through when this issue occurred** -- I was unable to reproduce the issue on my end.
>
>    1. Ensure the `Offer to save Passwords` and `Auto-sign in` options are enabled in `Settings --> Passwords --> Autofill`
>    2. Visit a site that requires login credentials (ie community.brave.com)
>    3. Enter credentials, ensure the browser prompts you to save the password for this site
>    4. Check `Settings --> Autofill --> Passwords` again to verify login data was saved
>
> ## Actual result:
> 
> Login data is not saved and, subsequently, the form fields will not autofill login data when returning to the site.
>
>
> ## Expected result:
> Autofill/password data should be saved and viewable in `Settings --> Autofill --> Passwords` and should also be entered into form fields where this data was saved.
>
> ## Reproduces how often:
> 
> For users encountering the issue is always present.
>
> ## Brave version (brave://version info)
> 
> Issues reported while running:
> - v0.70.121
> - v0.70.122
>
> ## Version/Channel Information:
> 
>
> - Can you reproduce this issue with the current release? Yes
> - Can you reproduce this issue with the beta channel? Yes
> - Can you reproduce this issue with the dev channel? Not enough info
> - Can you reproduce this issue with the nightly channel? Not enough info
>
> ## Other Additional Information:
>
> - Does the issue resolve itself when disabling Brave Shields? No
> - Does the issue resolve itself when disabling Brave Rewards? No
> - Is the issue reproducible on the latest version of Chrome? No
>
> ## Miscellaneous Information:
> 
> - https://community.brave.com/t/passwords-and-usernames-will-not-save-in-the-latest-desktop-build-version-0-70-121-chromium-78-0-3904-70/89820
>
>    - This user was on macOS Catalina 10.15.1
>    - This user was previously able to save login data as intended until they updated to v0.70.121
>    - This user gets the same results in Brave Beta build as well
>    - Note that for this user `Login data` file **was not found** in `~/Application Support/BraveSoftware/Brave-Browser/[Default or named Profile folder]`
>
> - https://community.brave.com/t/lost-all-brave-passwords-on-update/90589
>    - This user was using macOs 10.14.4
>    - This user had previous login data saved in browser without issue
>    - This user claims that after updating to v0.70.122, password/autofill data was lost
>
> - https://community.brave.com/t/brave-doesnt-remember-any-of-my-passwords/87646
>    - This user recently download the browser and is unable to save any passwords
>    - User stopped responding, exact OS version is unknown, but using macOS
>
> - https://community.brave.com/t/brave-does-not-save-passwords/89889/6
>    - This user is on macOS v10.12.6
>    - This user gets the same results in Brave Beta  build as well
>    - This user provided detailed information:
> > I originally installed the older version of Brave on a user account that does not have admin rights.
> > On that account, Brave did save passwords as expected. At some point, Brave started running out of memory regularly and crashed every day.
> > I continued using Brave without update on another user account (with admin rights).
> > There, Brave did not have the crashing problem, but it never saved website login passwords on that account.
> > Yesterday I installed the latest version of Brave on the user account with admin rights. Unfortunately, the new version of Brave does not save website login passwords either – at least not on the user account on which I installed it (this is the account with admin rights. The original older version of Brave was initially installed on the other user account, without admin rights).
> >
> > I examined the unix rights of Braves files in the Library. The directories have read/write permissions for the account where I am using Brave (old and new version. old version installed on other user account. new Brave version installed on this account).
> > So that does not provide any clue what is going wrong.
> > [...]
> >There is one thing that I noticed though:
> >I tried to save passwords for this community forum and 2 other forums.
> >Brave Beta did not remember the passwords for any of these.
> >BUT it did remember the user names for the other 2 web forums.
> >The user names for these other web forums were remembered correctly for login, but the text field for the passwords was empty. That is by the way the same with the official release (non-beta).
>
> - https://community.brave.com/t/passwords-not-working/89293/3
>    - This user is on macOS Catalina v10.5.1, Brave v0.70.121
>    - This user lost passwords and other browser settings after updating to Catalina
>    - This user cannot save any login data -- does not appear in `Autofill`. This happens on any/all websites
>
> - https://community.brave.com/t/lost-all-passwords-history-bookmarks-etc-upon-app-re-launch/89336
>    - This user is on macOS High Sierra v10.13.6, Brave v0.70.121
>    - This user claims their browsing data was lost including all logins/passwords after a browser relaunch.
>    - They also note that data from _all their profiles_ was removed or is not appearing.

**Comment by slayer** [comment]
The same issue:

Ubuntu  eoan
Brave about:
Version 1.0.0 Chromium: 78.0.3904.97 (Official Build) (64-bit)

```
ii  brave-browser                         1.0.0                               amd64        The web browser from Brave
ii  brave-keyring                         1.5                                 all          Brave Browser keyring and repository files

```

**Comment by cjgb** [comment]
I experienced the same problem after reinstalling my OS (a change of Linux distros): passwords were not stored even if I asked Brave to do so. The list of stored passwords under `Settings > Passwords` was empty even right after having tried to store one.
I could only _fix_ it by deleting the `Login Data` SQLite database inside my profile. In my Linux, it is located at `~/.config/BraveSoftware/Brave-Browser/Default/Login Data`.
Needless to say, I lost all the passwords I had stored previously.

**Comment by millerlarson** [comment]
> I could only _fix_ it by deleting the `Login Data` SQLite database inside my profile.

Thank you for this.

I was hitting this in OSX, and was able to correct the issue by nuking 'Login Data' in: `~/Library/Application Support/BraveSoftware/Brave-Browser/Default/`

**Comment by tkserver** [comment]
@millerlarson yes yes!  Thank you!  I've been very irritated by this problem and your solution worked!

**Comment by MrBrownstone** [comment]
> > I could only _fix_ it by deleting the `Login Data` SQLite database inside my profile.
> 
> Thank you for this.
> 
> I was hitting this in OSX, and was able to correct the issue by nuking 'Login Data' in: `~/Library/Application Support/BraveSoftware/Brave-Browser/Default/`

Worked for me!!

**Comment by tincho** [comment]
Hello!

I'm running Ubuntu 

Brave Version: 1.9.72 Chromium: 81.0.4044.138 (Official Build) (64-bit)

Passwords *are* saved but I can't *see* them

**Comment by simonhong** [comment]
I think this is duplicated issue with https://github.com/brave/brave-browser/issues/3196.
and fix was merged today.

**Comment by simonhong** [comment]
Closed as this is duplicated with #3196
